lib/dfe/analytics/entity_table_check_job.rb

```diff
@@ -27,16 +27,47 @@ def perform
 
       def entity_table_check_data(model)
         checksum_calculated_at = Time.now.in_time_zone(TIME_ZONE).iso8601(6)
```

Comment:
Just had a thought on `calculated_at`. Should we get the current timestamp from the database as the server timestamp and postgres timestamp could be out of sync ?

Reply:
👍 Good plan, now updated

Reply:
👍  now updated